cmd/antrea-agent/agent.go

```diff
@@ -876,7 +876,7 @@ func run(o *Options) error {
 			klog.InfoS("Waiting for AntreaProxy to be ready")
 			if err := wait.PollUntilContextCancel(wait.ContextForChannel(stopCh), time.Second, false, func(ctx context.Context) (bool, error) {
 				klog.V(2).InfoS("Checking if AntreaProxy is ready")
-				return proxier.GetProxyProvider().SyncedOnce(), nil
+				return proxyServer.GetProxier().GetProxyProvider().SyncedOnce(), nil
```

Comment:
this seems unnecessary complicated. Should we add a `SyncedOnce` method to `proxyServer` or ` `proxyServer.GetProxier()` for simplicity?